src/test/scala/org/constellation/cluster/E2ETest.scala

```diff
@@ -26,7 +27,7 @@ class E2ETest extends AsyncFlatSpecLike with Matchers with BeforeAndAfterAll wit
   override def beforeAll(): Unit = {
     // Cleanup DBs
     //Try{File(tmpDir).delete()}
-    //Try{new java.io.File(tmpDir).mkdirs()}
+    Try{new java.io.File(tmpDir).mkdirs()}
```

Comment:
can you use better files File here? Trying to stay consistent with it.

Reply:
Yeah, theres some other calls I replaced out of paranoia for the file handle open thing that we can change back too now re: snapshots.